src/main/java/com/alkemy/wallet/entity/FixedTermDeposit_Joaquin.java

```diff
@@ -0,0 +1,41 @@
+package com.alkemy.wallet.entity;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import javax.persistence.*;
+import java.sql.Timestamp;
+import java.util.UUID;
+
+@Entity
+@Getter
+@Setter
+@Table(name = "fixed_term_deposit", schema = "JAVHASW1")
+public class FixedTermDeposit_Joaquin {
+
+    @Id
+    @Column(name = "ID_FIXED_TERM_DEPOSIT", nullable = false)
+    @OneToMany(fetch = FetchType.EAGER, cascade = CascadeType.ALL)
```

Comment:
Por favor verificar la relación, debería ser Many to One

Reply:
Corregido.